modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Text/Text.es.js

```diff
@@ -25,6 +25,19 @@ import {FieldBase} from '../FieldBase/ReactFieldBase.es';
 import {useSyncValue} from '../hooks/useSyncValue.es';
 import withConfirmationField from '../util/withConfirmationField.es';
 
+const getTooltipTitle = (placeholder, value) => {
+	let tooltipText = '';
+
+	if (value !== '') {
+		tooltipText = value;
+	}
+	else if (placeholder !== '') {
+		tooltipText = placeholder;
+	}
+
+	return tooltipText !== '' ? {title: tooltipText} : '';
+};
+
```

Comment:
Same as above. I am starting to think this should extracted to a `utils.js` file. What do you think?